### PR TITLE
Update per Feb. 7th meeting

### PR DESCRIPTION
**Pull request by mjkoster**
Removed the 2.07 code for no data and expect publisher to publish
something. Clean up Max-Age so it applies to the topic. address email
comments.

### CODE REVIEW

File: draft-ietf-core-pubsub.md [hunk 1]
```diff
@@ -195,19 +197,32 @@ object in a publish-subscribe system. Topics are conventionally formed
...
-
+is implemented using URIs. The full URI of a topic includes a scheme and 
+authority for the Broker, for example 
+"coaps://10.0.0.13:5684/EP-33543/sen/3303/0/5700".
+
+A Topic may have a lifetime defined by using the CoAP Max-Age option on topic 
+creation, or on publish operations to the topic. The lifetime is refreshed each 
+time a representation is published to the topic. If the lifetime expires, the 
+topic is removed from discovery responses, returns errors on subscription, and 
+any outstanding subscriptions are cancelled.
```
Comment (akeranen) [inline draft-ietf-core-pubsub.md]: s/outstanding subscriptions are cancelled/outstanding subscriptions for the topic are cancelled by sending a "4.04" response code to subscribers of the topic/

File: draft-ietf-core-pubsub.md [hunk 2]
```diff
@@ -410,29 +425,29 @@ Ony one level in the topic hierarchy may be created as a result of a CREATE
 operation, unless create on PUBLISH is supported (see {{sec-publish}}).
 The topic string used in the link target MUST NOT contain the "/" character.
 
-A topic creator MUST include exactly one content format link attribute value (ct)
-in the create payload. If the Broker does not support the indicated format for
-both publish and subscribe, or if there is more than one "ct" value included in
-the request, the Broker MUST reject the operation with an error code of
-"4.00 Bad Request".
+A topic creator MUST include one or more content format link attribute values (ct) in the create payload. If the Broker does not support the indicated formats for both publish and subscribe, the Broker MUST reject the operation with an error code of "4.00 Bad Request".
```
Comment (akeranen) [inline draft-ietf-core-pubsub.md]: Supporting just single ct is better.

File: draft-ietf-core-pubsub.md (hunk 1 above)
Comment (mjkoster) [reply draft-ietf-core-pubsub.md]: This part needs WG discussion

File: draft-ietf-core-pubsub.md (hunk 2 above)
Comment (mjkoster) [reply draft-ietf-core-pubsub.md]: This whole feature needs WG discussion

Comment (mjkoster) [reply draft-ietf-core-pubsub.md]: Updated to require one ct

**Pull request merged**